### PR TITLE
change EndPort type to *int32

### PR DESCRIPTION
**Pull request by l8huang**
This change is for aligning type with NetworkPolicyPort, this could make the conversion from MultiNetworkPolicy to NetworkPolicyPort easier.

### CODE REVIEW

**Comment by l8huang** [comment]
@s1061123 could you please take a look?

File: pkg/apis/k8s.cni.cncf.io/v1beta2/types.go
Comment (s1061123) [inline pkg/apis/k8s.cni.cncf.io/v1beta2/types.go]: Why do you need the changes? I suppose we don't need that.

**Review by s1061123 — CHANGES_REQUESTED** [review]

**Comment by s1061123** [comment]
Hi @l8huang, thank you so much for the PR. Could you please take a look into my comment?
Almost Okey, so I'll approve once you incorporate my comment.

File: pkg/apis/k8s.cni.cncf.io/v1beta2/types.go
Comment (l8huang) [reply pkg/apis/k8s.cni.cncf.io/v1beta2/types.go]: hmmm, I use VS Code, when I save the file, it invokes some tools(probably gofmt) to do post-processing, which cause this change.

Comment (l8huang) [reply pkg/apis/k8s.cni.cncf.io/v1beta2/types.go]: removed this change, PTAL, thanks

**Comment by l8huang** [comment]
Hi @s1061123 , thanks for looking at this PR, your comment addressed, could you please take a look?

**Comment by l8huang** [comment]
@s1061123 please kindly let know if it's okay to you? Thanks.

**Comment by s1061123** [comment]
@l8huang sorry for the late reply. It looks good to me, so after fixing CI, I will merge that.

**Comment by l8huang** [comment]
@s1061123 Thanks for getting this PR merged.

I have some questions regard to versioning, created a issue for that: https://github.com/k8snetworkplumbingwg/multi-networkpolicy/issues/23. Could you please take a look?

**Pull request merged**
